### ISSUE TITLE
How do I get my native guide back?

### ISSUE THREAD

**Issue by brokman1** [files-community/Files#6876]
### Description

What needs to be changed in the registry so that the native explorer returns?

### Expected behavior

more stable work of the program

### Files Version

2.0.34

### Windows Version

Windows 10 Pro 21H1 19043.1348

**Comment by BanCrash** [comment]
Hi, you have to follow the instructions here to unset as default file explorer:

https://files.community/docs/configuring/replace-file-explorer

TL;DR: download [this zip](https://raw.githubusercontent.com/files-community/files-community.github.io/main/data/SetFilesAsDefault.zip) and run UnsetFilesAsDefault.reg to restore windows explorer

**Comment by brokman1** [comment]
> Hi, you have to follow the instructions here to unset as default file explorer:
> 
> https://files.community/docs/configuring/replace-file-explorer
> 
> TL;DR: download [this zip](https://raw.githubusercontent.com/files-community/files-community.github.io/main/data/SetFilesAsDefault.zip) and run UnsetFilesAsDefault.reg to restore windows explorer

Why can't this function be added to the program settings?

**Comment by BanCrash** [comment]
> > Hi, you have to follow the instructions here to unset as default file explorer:
> > https://files.community/docs/configuring/replace-file-explorer
> > TL;DR: download [this zip](https://raw.githubusercontent.com/files-community/files-community.github.io/main/data/SetFilesAsDefault.zip) and run UnsetFilesAsDefault.reg to restore windows explorer
> 
> Why can't this function be added to the program settings?

It's already on the Experimental part of Files settings, though previous insider versions had a bug that caused that "the option to reset the default file explorer wouldn't work after reinstalling Files". This bug has been fixed on last insider version.
